project_scrum/models/project_scrum_us.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api
+
+
+class ProjectScrumUs(models.Model):
+    _name = 'project.scrum.us'
+    _description = 'Project Scrum Use Stories'
+    _order = 'reference'
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+
+    @api.model
+    def create(self, vals):
+        vals['reference'] = self.env['ir.sequence'].next_by_code('user.story')
+        return super(ProjectScrumUs, self).create(vals)
+
+    @api.model
+    def _get_moscow_field(self):
```

Comment:
Return the values from project.task. No you define the moscow selection in two separate places.